app/models/category.rb

```diff
@@ -0,0 +1,16 @@
+class Category < ApplicationRecord
+  belongs_to :user
+  has_many :category_records
+  has_many :records, through: :category_records
+  has_one_attached :icon
+
+  validates :name, presence: true, length: { maximum: 250 }
+
```

Comment:
- [ ] I was able to create a category without an icon. However, this should not be allowed. Kindly add validation for the `:icon` when creating a category.